presto-hive/src/test/java/com/facebook/presto/hive/TestAriaHiveDistributedQueries.java

```diff
@@ -294,6 +294,7 @@ public void testFilters()
 
         assertQuery(ariaSession(), "SELECT shipdate FROM lineitem_aria WHERE long_decimal_discount < decimal '0.3'",
                 "SELECT shipdate FROM lineitem where discount < 0.3");
+        assertQuery(ariaSession(), "select comment from lineitem where comment = 'ep furiously above the pending de'");
```

Comment:
nit: SQL keywords need to be capitalized - "SELECT comment FROM lineitem WHERE comment = 'ep furiously above the pending de'"